.teamcity/settings.kts

```diff
@@ -48,6 +48,9 @@ val targets = arrayOf(
         "windows/amd64/1.18",
         "windows/amd64/tip",
 
+        "windows/arm64/1.18",
```

Comment:
IMO we should add these until there are build agents. I don't know if teamcity is going to complain but even if it doesn't there's no point on having them here.

Reply:
Related to https://github.com/go-delve/delve/pull/3063#discussion_r933114531.

This target is really running on amd64 and cross-compiling to arm64.